flags.go

```diff
@@ -37,25 +41,26 @@ type jsonFileBackend struct {
 	filename string
 }
 
-func readFile(file string, backend string, parse func(io.Reader) (map[string]Flag, error)) (map[string]Flag, error) {
+func readFile(file string, backend string, parse func(io.Reader) (map[string]Flag, time.Time, error)) (map[string]Flag, time.Time, error) {
 	var checkStatus statsd.ServiceCheckStatus
 	defer func() {
 		stats.SimpleServiceCheck("goforit.refreshFlags."+backend+"FileBackend.present", checkStatus)
 	}()
 	f, err := os.Open(file)
 	if err != nil {
 		checkStatus = statsd.Warn
-		return nil, err
+		log.Print("unable to open backend file:: ", err)
```

Comment:
This looks a bit odd. There should be just one colon, and there should be a newline. Maybe add a reference to "goforit" somewhere too :)